generators/root/templates/base/codemagic.yaml

```diff
@@ -0,0 +1 @@
+workflows: []
```

Comment:
Should we really add a codemagic config files in all projects event if there is no mobile app ? It could be confusing for users that get this file when not needed.

Reply:
This will be fixed by https://github.com/thetribeio/generator-project/pull/806 with the additional benefit that projects with only a mobile app won't get a useless CircleCI config file.